src/graphql.rs

```diff
@@ -522,6 +522,9 @@ struct MockAgentManager {}
 #[cfg(test)]
 #[async_trait::async_trait]
 impl AgentManager for MockAgentManager {
+    async fn broadcast_trusted_domains(&self) -> Result<(), anyhow::Error> {
+        Ok(())
```

Comment:
The `broadcast_trusted_domains` function in `MockAgentManager` has been modified to return `Ok(())` for the purpose of executing test code